arbitrum-docs/for-devs/concepts/differences-between-arbitrum-ethereum/rpc-methods.mdx

```diff
@@ -0,0 +1,133 @@
+| `l1BlockNumber` | The L1 block number that would be used for `block.number` calls. More information in [Block numbers and time](/for-devs/concepts/differences-between-arbitrum-ethereum/block-numbers-and-time.mdx) |
+| `gasUsedForL1`  | Amount of gas spent on L1 calldata in units of L2 gas. More information in [Gas and fees](/arbos/gas.mdx)                                                                                          |
+
+## Blocks
+
+When calling [`eth_getBlockByHash`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getblockbyhash) and other methods that return a block, Arbitrum includes a few additional fields and leverages some existing fields in different ways than Ethereum.
+
+### Additional fields
+
+On RPC calls that return a block, the following fields are added to the returned object.
+
+| Field name      | Description                                                                                                                                                                                          |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `l1BlockNumber` | An approximate L1 block number that occurred before this L2 block. More information in [Block numbers and time](/for-devs/concepts/differences-between-arbitrum-ethereum/block-numbers-and-time.mdx) |
+| `sendCount`     | The number of L2 to L1 messages since Nitro genesis                                                                                                                                                  |
+| `sendRoot`      | The Merkle root of the outbox tree state                                                                                                                                                             |
+
+### Existing fields with different behavior
+
+On RPC calls that return a block, the following fields will have a different content than what's received on Ethereum.
+
+| Field name   | Description                                                                                                           |
+| ------------ | --------------------------------------------------------------------------------------------------------------------- |
+| `extraData`  | This field is equivalent to `sendRoot`                                                                                |
+| `mixHash`    | First 8 bytes is equivalent to `sendCount`, second 8 bytes is equivalent to `l1BlockNumber`                           |
+| `difficulty` | Fixed at `0x1`                                                                                                        |
+| `gasLimit`   | Value is fixed at `0x4000000000000`, but it's important to note that Arbitrum currently has a 32M gas limit per block |
```

Comment:
Specify Arbitrum _One_ has a 32 mil gas limit (maybe also link to https://docs.arbitrum.io/for-devs/chain-params for the others)